manifests/charts/ztunnel/templates/rbac.yaml

```diff
@@ -20,7 +20,7 @@ metadata:
     {{- .Values.annotations | toYaml | nindent 4 }}
 {{- end }}
 ---
-{{- if (eq .Values.platform "openshift") }}
+{{- if (eq .Values.global.platform "openshift") }}
```

Comment:
(for context to observers: `platform` has been a `global` for some time before this, but we were munging globals just for `ztunnel` previously, so the `global`-ness was being hidden - this is not newly-globalizing `platform`)